apps/coscrad-frontend/src/components/resources/terms/term-detail.full-view.presenter.tsx

```diff
@@ -6,21 +6,44 @@ import {
 } from '@coscrad/api-interfaces';
 import { AudioClipPlayer } from '@coscrad/media-player';
 import { Box } from '@mui/material';
+import { useContext } from 'react';
+import { ConfigurableContentContext } from '../../../configurable-front-matter/configurable-content-provider';
 import { ResourceDetailFullViewPresenter } from '../../../utils/generic-components/';
 import { buildDataAttributeForAggregateDetailComponent } from '../../../utils/generic-components/presenters/detail-views/build-data-attribute-for-aggregate-detail-component';
+import { ResourceNamePresenterProps } from '../../../utils/generic-components/presenters/detail-views/resource-detail-presenter-header';
+import { FlatMultilingualTextPresenter } from '../../../utils/generic-components/presenters/flat-multilingual-text-presenter';
+import { getTextItemsForMultilingualTextPresenter } from '../../../utils/generic-components/presenters/get-text-items-for-multilingual-text-presenter';
+
+const TermNamePresenter = ({ name, variant }: ResourceNamePresenterProps): JSX.Element => {
+    const { defaultLanguageCode } = useContext(ConfigurableContentContext);
+
+    const { primaryMultilingualTextItem, translations } = getTextItemsForMultilingualTextPresenter(
+        name,
+        defaultLanguageCode
+    );
+
+    return (
+        <FlatMultilingualTextPresenter
```

Comment:
Don't we want the accordion for Terms? We've gotten positive feedback on the "flash card" experience it provides (only for terms though).

Reply:
Sounds like we might need to make this configurable.